.vscode/tasks.json

```diff
@@ -0,0 +1,47 @@
+{
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "label": "Aristotle: Prove Current File",
+      "type": "shell",
+      "command": "bash",
+      "args": [
+        "-l",
+        "-c",
+        "python3 ${workspaceFolder}/scripts/aristotle/prove.py '${file}'"
+      ],
```

Comment:
Why invoke `bash` rather than invoking `python3` directly? This is going to result in quoting issues as written